### ISSUE TITLE
[Feature Request] Show filter categories on Installed tab as well

### ISSUE THREAD

**Issue by jcperil** [Iamlooker/Droid-ify#189]
There are categories on the Explore tab like Automaton, Connectivity, Food, Games, etc., that allows you to filter apps accordingly.

![filter](https://user-images.githubusercontent.com/89805719/205624194-0a9891ca-3a91-4d7b-969e-f01a0aef2ed3.jpg)

This is a request to add these filtering categories to the Installed tab as well.

**Comment by Minimalx** [comment]
Hi! I am a user who likes minimalism.

 I think the categories supposed to help you find new apps to install. Adding it to the Installed tab sound kinda useless to me.
 I don't see this requested feature being commonly used by most people.

 I am against adding features with niche functionalities.

 I just wanted to comment my opinion, at the end it will be the dev's decision.

**Comment by jcperil** [comment]
It is certainly possible that this request is uncommon. I would mainly like to see the apps that I installed from a different repo.

Filtering based on **topics** and based on **repositories** are all together. Maybe there can be a repository filter in the Installed tab. It is not totally useless.

**Comment by vertigo220** [comment]
I'm mixed about this. OTOH, it seems like something I might find useful on rare occasion, like if I want to see all the different apps of a certain type I have installed, but OTOH if I wanted to do that I could just look in my launcher or app organizer, though that wouldn't tell what repo it's from. Ultimately, all it would be doing is taking a feature already there and copying it to another tab, so likely very little development work necessary and very minor change in the UI, so not really adding clutter or complexity. Out of curiosity, why are you interested in seeing what apps are from what repos? The only use case for this I've thought of before was to see what apps are also in the Play Store, so I could uninstall them from there and install from Droid-ify, but despite the fact that, to my knowledge, the two installs used to be incompatible, I've found that apps in both can be updated in either.
